src/frontend/apps/e2e/__tests__/app-desk/member-create.spec.ts

```diff
@@ -25,7 +25,7 @@ test.describe('Members Create', () => {
     await expect(page.getByRole('button', { name: 'Cancel' })).toBeVisible();
   });
 
-  test('it selects 2 users', async ({ page, browserName }) => {
+  test('it selects 2 users and 1 invitation', async ({ page, browserName }) => {
```

Comment:
is this test a bit redundant with the one `it sends a new invitation and adds a new member`

Reply:
Yes it is a bit redundant in the actions but I don't control the same things, it will be a big test if I had all the controls in the same one.